### ISSUE TITLE
Launchx1.0.1 seems to be invalid after firetv is automatically updated

### ISSUE THREAD

**Issue by gitcute** [codefaktor/FTVLaunchX#11]
There is no way to return to the atv interface by pressing the home button.

**Comment by MagicAndre1981** [comment]
this is caused by last FireOS update 6.2.7.3, click back button until you are on launcher or press home button and pin the launcher as favorite to Amazon Home to launch the launcher again

**Comment by oathzed** [comment]
New fix by Seen Wolf! https://forum.xda-developers.com/showpost.php?p=83468741&postcount=461

**Comment by ErikAbele** [comment]
Nice! TechDoctorUK also had the idea of disabling the Amazon launcher but I didn't find the time to even look at all the issues, it's on my list for tomorrow actually... am quite busy with other stuff :-/

**Comment by oathzed** [comment]
> Nice! TechDoctorUK also had the idea of disabling the Amazon launcher but I didn't find the time to even look at all the issues, it's on my list for tomorrow actually... am quite busy with other stuff :-/

Yeah I thought of that as well I just didn't know how to intercept it with the other launcher lol.

No problem man. Looking forward to when you can take a look at it. Curious on how you're going to implement it. And whether or not you can get "last used app" to launch on wake instead of just the launcher ;). That would be game changing, I think techdoctoruks app did that.. the old one anyway

**Comment by MagicAndre1981** [comment]
the settings app doesn't work for me on my FireTV Stick 4k

**Comment by oathzed** [comment]
> the settings app doesn't work for me on my FireTV Stick 4k

The settings app that you installed? What does work exactly? We need more info

**Comment by MagicAndre1981** [comment]
I've installed the settings app, started LeanOnFireLauncher and the settings app and selected to disable the amazon launcher, here I get error message about ADB. Adb is on and deployed the apk from my laptop, so adb works.

**Comment by oathzed** [comment]
> I've installed the settings app, started LeanOnFireLauncher and the settings app and selected to disable the amazon launcher, here I get error message about ADB. Adb is on and deployed the apk from my laptop, so adb works.

Do not connect anything to adb. I had the same issue. Disconnect your laptop from adb. Then turn off and on adb on the firetv.

Now open the launcher of your choice.
Open the settings app.
Click disable amazon launcher.
Now press home button

**Comment by oathzed** [comment]
> Nice! TechDoctorUK also had the idea of disabling the Amazon launcher but I didn't find the time to even look at all the issues, it's on my list for tomorrow actually... am quite busy with other stuff :-/

Did you end up getting anywhere with FTVLaunchx?
